src/inverters/brands/huawei.ts

```diff
@@ -1,6 +1,7 @@
 import {InverterSettingsDto, InverterStatus} from '../dto/inverter-settings.dto';
 import {InverterModel} from '../../types';
 import {localize} from '../../localize/localize';
+import {Utils} from '../../helpers/utils';
```

Comment:
What was the reason for importing this?

Reply:
I think it's unnecessary code. 

The new battery function on the base class was extended here to overwrite the function. However, I left it out for now and left the functionality in the index until I can test it properly. 

So this import is left over/unnecessary